tests/core/ckeditor/manual/hcdetection.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: feature, 4.15.2, 4379
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, clipboard, basicstyles, about, stylescombo, format, link, image
+
+1. Enable high contrast mode in your OS.
```

Comment:
This is a little inconsistent with expected - this step assumes one can enable HC mode and Expected/Unexpected mentions cases where HC mode is not supported by OS which makes it somewhat confusing.

At least we need better description or split this into two parts (OS supporting HC mode and the ones without it).

Reply:
Probably splitting test into two will be the most readable solution.